x-pack/plugin/otel-data/src/main/resources/index-templates/traces-otel@template.yaml

```diff
@@ -20,6 +20,10 @@ ignore_missing_component_templates:
   - traces@custom
   - traces-otel@custom
 template:
+  settings:
```

Comment:
There is no `traces-otel@settings` component template, so moved the synthetic source to the index template level. This index template is the only user of `traces-otel@mappings`.

Reply:
I think you can remove this entirely as `traces-otel@mappings` already configures `index.mode: logsdb`. Enabling synthetic _source was done before we added logsdb for this data stream.